dashboard/src/components/AppView/ChartInfo.tsx

```diff
@@ -1,28 +1,65 @@
 import * as React from "react";
+import { ArrowUpCircle, CheckCircle } from "react-feather";
+import { Link } from "react-router-dom";
 
 import { hapi } from "shared/hapi/release";
+import { IChartUpdate } from "shared/types";
 import Card, { CardContent, CardFooter, CardGrid, CardIcon } from "../Card";
 
 import placeholder from "../../placeholder.png";
 import "./ChartInfo.css";
 
 interface IChartInfoProps {
   app: hapi.release.Release;
+  update?: IChartUpdate;
 }
 
 class ChartInfo extends React.Component<IChartInfoProps> {
   public render() {
-    const { app } = this.props;
+    const { app, update } = this.props;
     const name = app.name;
     const metadata = app.chart && app.chart.metadata;
     const icon = metadata && metadata.icon;
     const iconSrc = icon ? icon : placeholder;
+    let updateStatusInfo: JSX.Element | null = null;
```

Comment:
consider moving this to a separate method, it avoids the need for initialising a variable as we can just use the return value